sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -395,6 +395,13 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class DelayStrategy
+    {
+        protected DelayStrategy() { }
```

Comment:
Thoughts on the naming here - DelayStrategy vs DelayPolicy vs something else?

/cc @christothes @annelo-msft

Reply:
I like the name as is

Reply:
I feel like `Policy` is largely taken, given we use those for our pipeline, but I like the way it describes the concept.  Strategy feels a little weird to me - is there any .NET/BCL naming precedent for "a thing that is a placeholder for an implementation"?  "Policy" is kind of that.

Reply:
Actually, this makes me curious - have we already said what the motivation for moving this into the public API is?  (Apologies if I'm forgetting)

Reply:
Yeah this is covered here - https://github.com/Azure/azure-sdk/issues/3841#issuecomment-1029433201